extensions/cornerstone-dicom-sr/src/commandsModule.ts

```diff
@@ -42,8 +43,57 @@ const _generateReport = (measurementData, additionalFindingTypes, options = {})
 
 const commandsModule = (props: withAppTypes) => {
   const { servicesManager } = props;
-  const { customizationService } = servicesManager.services;
+  const { customizationService, measurementService, viewportGridService, uiDialogService } =
+    servicesManager.services;
   const actions = {
+    jumpToMeasurement: ({ uid }) => {
```

Comment:
These don't belong here; they are generic measurement manipulations (jump, remove, rename) and should be in the cornerstone extension.